yapapi/runner/events.py

```diff
@@ -148,23 +147,28 @@ class WorkerFinished(AgreementEvent):
 
 
 @dataclass
-class ScriptSent(AgreementEvent, TaskEvent):
+class ScriptEvent(AgreementEvent):
```

Comment:
You may use `@dataclass(init=False)` here to make sure (at runtime) that it's not possible to instantiate `ScriptEvent`.

Reply:
done.